workflows/sarek.nf

```diff
@@ -628,6 +627,22 @@ workflow SAREK {
         // and remove patient ID field & null value for further processing [meta1, [cram1,crai1]] [meta2, [cram2,crai2]]
         cram_variant_calling_tumor_only = cram_variant_calling_tumor_filtered.transpose().map{ it -> [it[1], it[2], it[3]] }
 
+        if(params.skip_germline_for_somatic){
```

Comment:
not too sure about the parameter name, I was also considering `skip_germline_for_paired` or something along those lines

Reply:
`only_paired_variant_calling`?

Reply:
nice, this is really says what it does